tailwind.config.js

```diff
@@ -1,8 +1,27 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ["./src/**/*.{html,ts}"],
+  content: ['./src/**/*.{html,ts}'],
   theme: {
-    extend: {},
+    extend: {
+      colors: {
+        primary: '#222328',
+        secondary: '#969AB0',
+        accent: '#3545E9',
+        warning: '#FEFD54',
+        success: '#63C995',
+        error: '#E23D69',
+        bright: '#FFFFFF',
+        muted: '#7C9CBF',
+        shade: '#DBE2EA',
+      },
+    },
+    fontFamily: {
+      sans: ['IBM Plex Sans', 'ui-sans-serif', 'system-ui'],
+    },
+    fontSize: {
+      'h1-title': ['72px', { lineHeight: '86px', fontWeight: '600' }],
+      'h3-title': ['56px', { lineHeight: '68px', fontWeight: '300' }],
```

Comment:
I do not really like such an approach. If thinking about styling beyond the pet project, such an approach is not scalable and flexible. Let's say in a real project you need h3-title but with fontWeight = 600, you need to add new rule here, and you should do that for any variations of font-size, line-height, font-weight.

I'd rather keep config simple, if you'd like to keep this approach, you can consider adding a component wrapper, that takes as an input type of typography (h1,h3, etc.), and depending on the type this component wrapper applies heeded font-size, line-height, etc as default styles. In case of any variations: you can add input customStyle (or any rather name for it) where you can provide any changes (this covers cases when you need, for example, h3 but with font-size 600)

Reply:
resolved